models/Movie.model.js

```diff
@@ -0,0 +1,20 @@
+const { mongoose, Schema, model } = require('mongoose');
+const Celebrity = require("../models/Celebrity.model");
+
+const movieSchema = new Schema(
+  {
+    title: String,
+    genre: String,
+    plot: String,
+    cast: {
```

Comment:
Here in the `cast` you probably want to wrap that in an Array, so you could pass multiple celebreties for the movies